type-definitions/hold.d.ts

```diff
@@ -1,6 +1,6 @@
 import {Stream, Source, Sink, Scheduler, Disposable} from "most";
 
-export default function hold<A>(stream: Stream<A>): Stream<A>;
+export function hold<A>(stream: Stream<A>): Stream<A>;
 
 export class Hold<T> implements Source<T> {
```

Comment:
Should we un-export this now?  There's certainly no harm in exporting it, but I dunno if it's useful.  Thoughts?